lib/schedule.js

```diff
@@ -165,7 +164,7 @@ function Job(name, job, callback) {
   };
 }
 
-util.inherits(Job, events.EventEmitter);
+Object.setPrototypeOf(Job.prototype, events.EventEmitter.prototype);
```

Comment:
Since we are targetting Node 6 now, wouldn't it be better to just use classes and do an `extend`?

Reply:
Actually, no, your solution is better: https://stackoverflow.com/questions/37194202/ie11-gives-script1002-error-when-defining-class-in-javascript

IE 11 is still a thing, so we can't use classes.

Reply:
I'd say so. That's probably a change with a larger scope and impact than this one, though.